app/api/exports.py

```diff
@@ -65,6 +65,28 @@ def export_download(event_id, path):
     return response
 
 
+@export_routes.route('/events/<string:event_identifier>/export/xcal', methods=['GET'])
```

Comment:
Is there any difference between event_identifier and event_id?

Reply:
Yes, in terms of the database id is the auto-generated integer key and event_identifier and identifier is the string hash i think

Reply:
Front-end uses the string hash(event_identifier) everywhere